Dockerfile

```diff
@@ -10,10 +10,13 @@ RUN export DEBCONF_NONINTERACTIVE_SEEN=true \
     apt-get -yyqq install ca-certificates ; \
```

Comment:
We could just add `setcap` to this install list, rather than pulling in a whole second Debian layer into the image.

Reply:
good call, lets do that.